src/lsg/patterns/view-switch/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { colors } from '../colors';
+import { Size } from '../copy';
+import { Icon, IconName, IconProps, Size as IconSize } from '../icons';
+import * as React from 'react';
+import { getSpace, Size as SpaceSize } from '../space';
+import styled from 'styled-components';
+
+export interface ViewSwitchProps {
+	fontSize?: Size;
+	justify?: 'start' | 'center' | 'end' | 'stretch';
```

Comment:
This is not very nice. You are just moving every css possibility in to the props.

Reply:
Well, that is right. When more items come in chrome we will be going to need them :).
Would you like me to reduce them to the current necessary?

Reply:
I think for now we should reduce them :)